src/player/Player.php

```diff
@@ -463,7 +463,12 @@ public function getFlySpeed() : float{
 		return $this->flySpeed;
 	}
 
-	public function setWalkSpeed(float $walkSpeed = 0.1) : void{
+	public function resetFlySpeed(): void
+	{
+		$this->setFlySpeed(0.05);
```

Comment:
I think const are better than hard code

Reply:
+1 for this. This would also remove the need for resetFlySpeed(), since you would be able to just do `setFlySpeed(DEFAULT_FLY_SPEED)` or similar.